prime-router/docs/releases/2021-07-22-release-notes.md

```diff
@@ -0,0 +1,28 @@
+#  ReportStream Release Notes
+
+*July 22, 2021*
+
+## General useful links:
+
+- All Schemas are documented here:  [Link to detailed schema dictionaries](../schema_documentation)
+- The Hub API is documented here: [Hub OpenApi Spec](../openapi.yml)
+- [Click here for all release notes](../releases)
+
+## For this release
+
+### Changes to api/reports response
+
+This release adds a new `routing` element to the Json response if the query string contains a `verbose=true` parameter. This routing shows the destination(s) for each item in the report along with the report index and trackingId. The openapi.yml has been updated to reflect these changes with the structure of the routing element. An example of this routing element:
```

Comment:
Perhaps add an example URL here for those that do not understand what a query is.